content/efcore/getting-started/installing-ef-core.md

```diff
@@ -1,58 +1,80 @@
-# Installing EF Core  
+# Installing EF Core 
  
-In this lesson, you will learn how to install Entity Framework Core (EF Core) on your machine.
+In this lesson, you will learn how to install Entity Framework Core on your machine.
```

Comment:
We reference it as EF Core later (heading), so we still need to specify the acronymn.